network_wrangler/roadwaynetwork.py

```diff
@@ -1284,183 +1572,505 @@ def validate_properties(
         Returns: boolean value as to whether the properties dictonary is valid.
         """
 
-        validation_error_message = []
-
+        valid = True
+        validation_error_message = ""
```

Comment:
changing this to string does not work with line 1596 below - `validation_error_message.append()`, in addition, this variable is not being used in any logging.